Manual_for_achitecture_browser/index.asciidoc

```diff
@@ -0,0 +1,62 @@
+= Manual for architecture browser
+These content help you to know how you can contribute to the content of the architecture browser.
+
+
+== Fetch upstream or fork repository
+If the repository already exists in your account, it will eventually become out of date, you have to make the changes that were made on the main repository. To do that, you have to click on "Fecht upstream " and choose "Fecht and merge".
+
+image::./addManual_image/fecht.png[fecht]
+But if it doesn't exist, then you have to fork it, so that the main repository will be generated in your own account. To do that, please click on the icon "Fork" from the main repository.
+
+image::./addManual_image/fork.png[Fork_Repository,100,100]
+
+== Open the repository on Gitpod
+The repository is already on your account. 
+Gitpod is already configured so that origin remote is not the main repository but your fork.
+You have only to go in the Readme.md and click on "https://gitpod.io/from-referrer/ ". This unique URL will automatically detect which repository it was visited from, and open that in Gitpod (using the Referrer HTTP header).
+
+
+== Create or Change tutorial
+You have the possibility to add a new tutorial by creating a new tutorial, you can also change the content of existing tutorial.
+
+To create a new tutorial, you have to click on the icon "new File".
+
+image::./addManual_image/newFile.png[newFile] 
+Then give the name to your file and press "Enter".
+
+To change a tutorial, select this tutorial, change the content and save.
+
+=== Upload file
+To upload a file, you have to click on the menu button on the far left.
+
+image::./addManual_image/menu1.png[menu button] 
+Select "File", then choose "open File". Enter the name of your file and click "ok". The selected file will be directly opened in the editor.
+
+== Commit and Push
+First of all, you have to create your own branch, to do it, select the icon "source control".
+
+image::./addManual_image/sourceControl.png[source Control]
+Go on the icon "view and more action". 
+
+image::./addManual_image/menu.png[menu button] 
+
+Choose "branch" and select "create branch". Give the name of the branch and press "Enter" .
+
+Now you can already do a commit, for this you must first make sure that a change has been made. Select the icon "source control", go on the icon "view and more action" choose "commit", provide the commit message and press "Enter" to confirm.
+
+image::./addManual_image/commitMessage.png[commmit]
+You can push, by clicking again on the icon "source control", go on the icon "view and more action" choose "push".
+
+== Create a pull request
+After doing a push, go to your Git account and create a pull request by clicking on "compare & pull request".
+
+image::./addManual_image/pr.png[PR]
+That is important to accept CLA if not yet done.
+
+== Stack specific solution
```

Comment:
You are not explaining why stack-specify solutions exist, when to use them and how to create one.